src/main/java/thePackmaster/powers/energyandechopack/ReceptorPower.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.powers.energyandechopack;
+
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class ReceptorPower extends AbstractPackmasterPower {
```

Comment:
Is `ReceptorPower` an old version of DivineTouchPower? It looks like it's doing similar logic, but it doesn't have power strings or an icon for it, it doesn't look like anything applies it to the player. Should delete the file if that's the case.